### ISSUE TITLE
unneeded dependencies

### ISSUE THREAD

**Issue by ripienaar** [nats-io/ruby-nats#116]
I presume thin is needed for the old ruby based nats server, its certainly not needed for the client.

What's the plan with the old server code, thin is a huge dependency that brings in a bunch of other dependencies it would be really great to avoid having to carry those around for people only after the client?

**Comment by wallyqs** [comment]
Missed removing it... I'll take it out from the gemspec since indeed client does not require it. Thanks for raising the issue.

**Comment by ripienaar** [comment]
seems all the client needs is `em`

**Comment by wallyqs** [comment]
Will make a minor release where this deps are gone, thanks for bringing up the issue.

**Comment by wallyqs** [comment]
@ripienaar minor release which drops unrequired deps now [published](https://rubygems.org/gems/nats/versions/0.7.1)
